### PR TITLE
Add test cases for artifact deployers

### PR DESCRIPTION
**Pull request by ThishaniLucas**

### CODE REVIEW

File: modules/core/src/test/java/org/apache/synapse/deployers/EndpointDeployerTest.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
+package org.apache.synapse.deployers;
```
Comment (manoramahp) [inline modules/core/src/test/java/org/apache/synapse/deployers/EndpointDeployerTest.java]: License header is missing

File: modules/core/src/test/java/org/apache/synapse/deployers/MessageStoreDeployerTest.java [hunk 2]
```diff
@@ -0,0 +1,127 @@
+package org.apache.synapse.deployers;
```
Comment (manoramahp) [inline modules/core/src/test/java/org/apache/synapse/deployers/MessageStoreDeployerTest.java]: License header is missing

File: modules/core/src/test/java/org/apache/synapse/deployers/EndpointDeployerTest.java (hunk 1 above)
Comment (ThishaniLucas) [reply modules/core/src/test/java/org/apache/synapse/deployers/EndpointDeployerTest.java]: fixed

File: modules/core/src/test/java/org/apache/synapse/deployers/MessageStoreDeployerTest.java (hunk 2 above)
Comment (ThishaniLucas) [reply modules/core/src/test/java/org/apache/synapse/deployers/MessageStoreDeployerTest.java]: fixed

File: modules/core/src/test/java/org/apache/synapse/deployers/LocalEntryDeployerTest.java [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        cfgCtx.setAxisConfiguration(axisConfiguration);
+
+        localEntryDeployer.init(cfgCtx);
+        localEntryDeployer.deploySynapseArtifact(inputElement, "sampleFile", null);
+
+        String inputUpdateXML = "<localEntry key=\"TestEntryUpdated\" xmlns=\"http://ws.apache.org/ns/synapse\">0.1</localEntry>";
+
+        OMElement updatedElement = AXIOMUtil.stringToOM(inputUpdateXML);
+
+        String response = localEntryDeployer.updateSynapseArtifact(updatedElement, "sampleUpdateFile", "TestEntry", null);
```
Comment (mohanvive) [inline modules/core/src/test/java/org/apache/synapse/deployers/LocalEntryDeployerTest.java]: Don't have unnecessary line spaces when there is no any logic separation between code segments.

Comment (ThishaniLucas) [reply modules/core/src/test/java/org/apache/synapse/deployers/LocalEntryDeployerTest.java]: Fixed with https://github.com/wso2/wso2-synapse/pull/1105

**Pull request merged**